src/clr-angular/data/tree-view/tree.ts

```diff
@@ -14,15 +26,65 @@ import { TREE_FEATURES_PROVIDER, TreeFeaturesService } from './tree-features.ser
     <clr-recursive-children *ngIf="featuresService.recursion"
                             [children]="featuresService.recursion.root"></clr-recursive-children>
   `,
-  providers: [TREE_FEATURES_PROVIDER],
+  providers: [TREE_FEATURES_PROVIDER, TreeFocusManagerService],
+  host: {
+    '[attr.tabindex]': 'tabindex',
+    '[attr.role]': '"tree"',
+    '[attr.aria-multiselectable]': 'isMultiSelectable',
+  },
 })
-export class ClrTree<T> {
-  // This component can also be used just to declare providers once for trees with multiple root nodes.
+export class ClrTree<T> implements AfterContentInit, OnDestroy {
+  constructor(
+    public featuresService: TreeFeaturesService<T>,
+    public focusManager: TreeFocusManagerService<T>,
+    private el: ElementRef
+  ) {}
 
-  constructor(public featuresService: TreeFeaturesService<T>) {}
+  private subscriptions: Subscription[] = [];
 
   @Input('clrLazy')
   set lazy(value: boolean) {
     this.featuresService.eager = !value;
   }
+
+  tabindex = 0;
+
+  get isMultiSelectable() {
+    return this.featuresService.selectable && this.rootNodes.length > 0;
```

Comment:
`this.featuresService.selectable` was blowing up for me in certain cases (maybe lazy loaded recursive?). Could you make sure that this works in all cases?

Reply:
Sure. Will check them.

Reply:
I've tested it in all browsers, and I haven't experienced any issues here.